src/applications/check-in/utils/navigation/index.js

```diff
@@ -59,8 +61,12 @@ const updateFormPages = (
     URLS.TRAVEL_MILEAGE,
   ];
 
-  // Skip travel pay if not enabled or if veteran has more than one appoinement for the day.
-  if (!isTravelReimbursementEnabled || appointments.length > 1) {
+  // Skip travel pay if not enabled, if veteran has more than one appointment for the day, or station if not in the allow list.
+  if (
+    !isTravelReimbursementEnabled ||
+    appointments.length > 1 ||
+    !travelAllowList.includes(appointments[0].stationNo)
```

Comment:
This works for now but should we loop through all the checkinable appointments so it's not dependent on the fact that we won't show the travel pay if there is more than one appointment?

Reply:
hmm, there could also be weird edge cases with appts at multiple stations if we ever start allowing travel pay with than one appointment... However I'd imagine that if we start doing multiple the allowList will already be long gone, but it might not hurt to leave ourselves a breadcrumb type comment.

Reply:
yeah the idea is to scope this to the specificity of our single appointment use case as it is today. Good idea to add a note though.